packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessments.js

```diff
@@ -317,6 +320,14 @@ const disabilityAssessments = [
 		score: SCORES.NON_INCLUSIVE,
 		feedbackFormat: redHarmful,
 	},
+	{
+		identifier: "dumbDown",
+		nonInclusivePhrases: [ "dumb down", "dumbing down", "dumbed down", "dumb it down", "dumbing it down",
+			"dumbed it down" ],
+		inclusiveAlternatives: "<i>oversimplify</i>",
+		score: SCORES.NON_INCLUSIVE,
+		feedbackFormat: redHarmful,
+	},
```

Comment:
You could consider splitting this up so that each form gets the alternative in a matching form of 'oversimplify'. For example 'dumbing down' -> 'oversimplifying' etc.